packages/playground/src/api/annotations/types.ts

```diff
@@ -24,7 +24,11 @@ export interface AnnotationModel extends AnnotationCreate {
   lastUpdatedTime: Date;
 }
 
-export interface AnnotationCreate {
```

Comment:
Outside the scope of this PR, but while you're at it :innocent: could you also fix `annotationType` in `AnnotationUpdate`.
Should be `annotationType?: SetField<AnnotationType>;`

Reply:
`AnnotationUpdate` is already like that :-) 
https://github.com/cognitedata/cognite-sdk-js/blob/master/packages/playground/src/api/annotations/types.ts#L46

However, there was another thing that was missing: `creatingAppVersion` in the `AnnotationFilterProps` which I added here: https://github.com/cognitedata/cognite-sdk-js/pull/806/commits/637e8c90c3ff8f70f919e14a25b428dab800d437

Reply:
hmm...shouldn't `annotationType?: SetField<AnnotationPayload>; ` -> `annotationType?: SetField<AnnotationType>; ` or am I missing something?

Reply:
Ah yes, good catch!
Fixed it here: https://github.com/cognitedata/cognite-sdk-js/pull/806/commits/f3c9c3e5d9e00344450a430c02680a665268ae91